reference/5-api-publisher/readme.md

```diff
@@ -1,9 +1,130 @@
+
+## Quick Start
+
+To demonstrate how the API Publisher works, this exercise copies all the data from [the sample hosted Ed-Fi ODS API](https://api.ed-fi.org) to a local sandbox Ed-Fi ODS API using the API client for the "minimal" template. (The database scripts are written for SQL Server.)
+
+### Configure Local Sandbox Environment
+
+Before using the API Publisher on a target ODS, you must create and configure an API client with the appropriate permissions for publishing.  The source and destination ODS/API must be of the same version.
+
+For Ed-Fi ODS / API 5.1 through 5.3 only: create and assign a claim set for the API Publisher by running the following database scripts:
+  * [Create-API-Publisher-Writer-Security-Metadata.sql](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Publisher/blob/main/eng/Create-API-Publisher-Writer-Security-Metadata.sql)
+  * [Configure-Minimal-Sandbox-Client-as-API-Publisher-Writer.sql](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Publisher/blob/main/eng/Configure-Minimal-Sandbox-Client-as-API-Publisher-Writer.sql)
+
+### Use the API Publisher
+
+The API Publisher has three options to use the product.  The API Publisher requires [.NET 8.0](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) to run:
+
+#### Option 1 - From binaries
+
+ 1. Download the latest published API Publisher package here:  [Ed-Fi API Publisher v1.0](https://dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_artifacts/feed/EdFi/NuGet/EdFi.ApiPublisher/overview/1.1.0).  Visit the page and click download.
+ 2. This will download a NuGet package to your computer.  Rename this file, `EdFi.ApiPublisher.1.1.0.nupkg`, to include .zip extension: `EdFi.ApiPublisher.1.1.0.zip`.
+ 3. The binary mentioned below is in the `EdFi.ApiPublisher.Win64` folder, as `EdFiApiPublisher.exe`.
+
+#### Option 2 - From Docker img
+
+The Docker image for the Ed-Fi API Publisher is available here: [Ed-Fi API Publisher v1.0 on Docker Hub](https://hub.docker.com/layers/edfialliance/ods-api-publisher/v1.1.0/images/sha256-23a6989b16d779d594692e1c9a8f2eac1790ed53312d424d85e869fec05eb914?context=explore).  Use this to include in your Docker environment and alongside other components of the Ed-Fi stack.
+
+#### Option 3 - Build the API Publisher from source code
+
+If you would like to build the API Publisher from source, build the solution by running the following command from the repository's root directory:
+
+`dotnet build`
+
+The API Publisher executable (`EdFiApiPublisher.exe`) will be located in the _.\EdFi.Tools.ApiPublisher.Cli\bin\Debug\net8.0_ subfolder.
+
+### Publish Data to Local Sandbox
+
+> **IMPORTANT: After changing security metadata for the API, YOU MUST RESTART the local sandbox Ed-Fi ODS API if it is already running.**
```

Comment:
Another opportunity for a Docusaurus-style admonition. I'm going to stop commenting on this, even though I see other examples below.

Reply:
fixed